config/main.cpp

```diff
@@ -27,12 +27,23 @@
 
 #include <LXQt/SingleApplication>
 
+#include <QCommandLineParser>
 #include "main_window.h"
 
 int main(int argc, char *argv[])
 {
     LXQt::SingleApplication a(argc, argv);
 
+    QCommandLineParser parser;
+    parser.setApplicationDescription(QStringLiteral("LXQt Config Globalkeys "));
+    const QString VERINFO = LXQT_GLOBALKEYS_VERSION \
```

Comment:
Why not - it it helps us to save two milliseconds at the program start - thats sums up in a year or two :D